src/database/standalone-sqlite.ts

```
@@ -3328,8 +3342,8 @@ if (!isMainThread) {
           parentPort?.postMessage(null);
           break;
         case 'saveBundle':
-          worker.saveBundle(args[0]);
-          parentPort?.postMessage(null);
+          const bundle = worker.saveBundle(args[0]);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)